content/pl/docs/home/_index.md

```diff
@@ -0,0 +1,58 @@
+---
+approvers:
+- chenopis
```

Comment:
Can you remove these lines? We want to use the reviewers/approvers from the OWNERS file instead. Only when the localization will be having _a lot_ of people, files can be appointed to specific contributors.

Reply:
Fixed.